File: launchdarkly/ruby/src/UpdateIntegrationConfigurationExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

patch_operation_1 = LaunchDarklyClient::PatchOperation.new
patch_operation_1.op = "replace"
patch_operation_1.path = "/on"

patch_operation = [
    patch_operation_1,
]

begin
    response = LaunchDarklyClient::IntegrationsBetaApi.new.update_integration_configuration(
        nil, # integration_configuration_id
        patch_operation,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling IntegrationsBetaApi#update_integration_configuration: #{e}"
end

File: launchdarkly/python/src/UpdateIntegrationConfigurationExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_operation_1 = models.PatchOperation(
        op="replace",
        path="/on",
    )

    patch_operation = [
        patch_operation_1,
    ]

    try:
        response = api.IntegrationsBetaApi(api_client).update_integration_configuration(
            integration_configuration_id="integrationConfigurationId_string",
            patch_operation=patch_operation,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling IntegrationsBetaApi#update_integration_configuration: %s\n" % e)
